packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -51,84 +51,297 @@ const WrappedCalendar = onClickOutside(Calendar)
 
 class DatePicker extends React.Component {
   static propTypes = {
+    /**
+     * Prop to change date when a valid date is selected.
+     */
     adjustDateOnChange: PropTypes.bool,
+    /**
+     * Prop to check for same dates.
+     */
     allowSameDay: PropTypes.bool,
+    /**
+     * Prop to auto complete date.
+     */
     autoComplete: PropTypes.string,
+    /**
+   * Prop to auto focus on date picker.
```

Comment:
1. Indentation should be corrected. 
2. Can be changed to "Prop to enable/disable auto focus on date picker".